### PR TITLE
Feature/chrome history extension

### PR DESCRIPTION
**Pull request by eefh**

### CODE REVIEW

**Comment by eefh** [comment]
Added Safari project

**Comment by posix4e** [comment]
Make sure to remove .ds_store and add to gitignore

**Comment by posix4e** [comment]
Do you wanna setup the github actions for ci/cd?

**Review by posix4e — CHANGES_REQUESTED** [review]
Haven’t looked in detail will look. In the meantime

- squash into single commit
- Use git lfs for the images 
- Why did you remove my readme? Change the todo boxes instead

File: Web Time Machine Extension/Resources/_locales/en/messages.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "extension_name": {
+        "message": "Web Time Machine Extension",
+        "description": "The display name for the extension."
+    },
+    "extension_description": {
```
Comment (posix4e) [inline Web Time Machine Extension/Resources/_locales/en/messages.json]: Perhaps you can help me with a Description

File: Web Time Machine Extension/Resources/background.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+browser.runtime.onMessage.addListener((request, sender, sendResponse) => {
+    console.log("Received request: ", request);
```
Comment (posix4e) [inline Web Time Machine Extension/Resources/background.js]: Probably don’t wanna leave debugging stuff around

File: Web Time Machine Extension/Resources/popup.js [hunk 3]
```diff
@@ -0,0 +1 @@
+console.log("Hello World!", browser);
```
Comment (posix4e) [inline Web Time Machine Extension/Resources/popup.js]: :(

File: Web Time Machine Extension/SafariWebExtensionHandler.swift [hunk 4]
```diff
@@ -0,0 +1,38 @@
+//
+//  SafariWebExtensionHandler.swift
```
Comment (posix4e) [inline Web Time Machine Extension/SafariWebExtensionHandler.swift]: Get rid of all the created by

File: Web Time Machine/Shared (App)/Resources/Script.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
+function show(platform, enabled, useSettingsInsteadOfPreferences) {
+    document.body.classList.add(`platform-${platform}`);
+
+    if (useSettingsInsteadOfPreferences) {
+        document.getElementsByClassName('platform-mac state-on')[0].innerText = "Web Time Machine’s extension is currently on. You can turn it off in the Extensions section of Safari Settings.";
```
Comment (posix4e) [inline Web Time Machine/Shared (App)/Resources/Script.js:5]: Is this where we show history?

File: README.md
Comment (posix4e) [inline README.md:1]: Please keep the todo for now

**Comment by posix4e** [comment]
I’m going to wait for you to tell me it’s ready to review

**Pull request closed without merging**